src/Init/Init_Load_FlagCriteria.cpp

```diff
@@ -140,76 +144,78 @@ void Init_Load_FlagCriteria()
-                                                                               &FlagTable_EngyDensity[lv][1] );
-            else if ( FlagMode == 4 )  sscanf( input_line, "%d%lf%lf%lf%lf%lf", &Trash, &FlagTable_Lohner[lv][0],
-                                                                                        &FlagTable_Lohner[lv][1],
-                                                                                        &FlagTable_Lohner[lv][2],
-                                                                                        &FlagTable_Lohner[lv][3],
-                                                                                        &FlagTable_Lohner[lv][4] );
-//          OPT__FLAG_USER has OPT__FLAG_USER_NUM columns to be loaded
-            else if ( FlagMode == 5 )
+            for (int t=0; t<OPT__FLAG_USER_NUM; t++)
             {
-               char *input_line0 = input_line;
-               int offset;
-
-               sscanf( input_line, "%d%n", &Trash, &offset );
+               sscanf( input_line, "%lf%n", &FlagTable_User[lv][t], &offset );
                input_line += offset;
-
-               for (int t=0; t<OPT__FLAG_USER_NUM; t++)
-               {
-                  sscanf( input_line, "%lf%n", &FlagTable_User[lv][t], &offset );
-                  input_line += offset;
-               }
-
-               input_line = input_line0;  // reset input_line
             }
-//          OPT__FLAG_NPAR_PATCH/CELL load integers
-            else if ( FlagMode == 6 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParPatch[lv] );
-            else if ( FlagMode == 7 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParCell [lv] );
-//          OPT__FLAG_ANGULAR has four columns to be loaded
-            else if ( FlagMode == 14 ) sscanf( input_line, "%d%lf%lf%lf", &Trash, &FlagTable_Angular[lv][0],
-                                                                                  &FlagTable_Angular[lv][1],
-                                                                                  &FlagTable_Angular[lv][2] );
-
-//          others use the default format: (integer, double)
-            else                       sscanf( input_line, "%d%lf", &Trash, &FlagTable[FlagMode][lv] );
+
+            input_line = input_line0;  // reset input_line
+         }
+//       OPT__FLAG_NPAR_PATCH/CELL load integers
+         else if ( FlagMode == 6 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParPatch[lv] );
+         else if ( FlagMode == 7 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParCell [lv] );
+//       OPT__FLAG_ANGULAR has three columns to be loaded
+         else if ( FlagMode == 14 )
+         {
+            sscanf( input_line, "%d%lf%lf", &Trash, &FlagTable_Angular[lv][0], &FlagTable_Angular[lv][1] );
+            if ( FlagTable_Angular[lv][0] < FlagTable_Angular[lv][1] )
+               Aux_Message( stderr, "WARNING : AngRes_Max < AngRes_Min at lv %d\n", lv );
```

Comment:
This check is applied only when both `AngRes_Min` and `AngRes_Max` are set, and check that `AngRes_Max` is smaller than `AngRes_Min`. Thus, the condition should be 
~~~ c
if ( FlagTable_Angular[lv][0] > 0.0   &&
     FlagTable_Angular[lv][1] > 0.0   &&
     FlagTable_Angular[lv][0] > FlagTable_Angular[lv][1] )
~~~

The warning message should be (also provide more information)
~~~ c
Aux_Message( stderr, "WARNING : AngRes_Max (%20.14e) > AngRes_Min (%20.14e) at lv %d\n", 
             FlagTable_Angular[lv][0] , FlagTable_Angular[lv][1], lv );
~~~